Composer/packages/lib/code-editor/src/lg/LgCodeEditor.tsx

```diff
@@ -273,11 +275,7 @@ export const LgCodeEditor = (props: LgCodeEditorProps) => {
             popExpandOptions?.onEditorPopToggle?.(false);
           }}
         >
-          <LgCodeEditor
-            {...omit(props, ['onNavigateToLgPage', 'popExpandOptions'])}
-            height={400}
-            onChange={onExpandedEditorChange}
-          />
+          <LgCodeEditor {...omit(props, ['popExpandOptions'])} height={400} onChange={onExpandedEditorChange} />
```

Comment:
if you add back onNavigateToLgPage to this larger editor, it will show a link below it, we dont want that link to be shown when user is editing in larger editor. At the same time you want to be able to go to definition using this callback. so you need to add a new prop to the component to toggle the link visibility on/off. Call it showTemplateDirectLink?: boolean. Add this prop to the conditions for showing the link:
```typescript
{showTemplateDirectLink && onNavigateToLgPage && lgOption && (
  <Stack horizontal tokens={templateLinkTokens} verticalAlign="center">
    <Text styles={grayTextStyle}>{formatMessage('Template name: ')}</Text>
    <LgTemplateLink as="button" styles={linkStyles} onClick={navigateToLgPage}>
      #{lgOption.templateId}()
    </LgTemplateLink>
  </Stack>
)}
```

![Screen Shot 2021-03-26 at 9 15 50 AM](https://user-images.githubusercontent.com/1483492/112661616-f58b4500-8e13-11eb-9e62-55c55e422ebe.png)

Reply:
nvm, I added it in another PR that is merged, you can now safely use it.